pymc/distributions/shape_utils.py

```diff
@@ -449,9 +449,10 @@ def convert_shape(shape: Shape) -> Optional[WeakShape]:
     """Process a user-provided shape variable into None or a valid shape object."""
     if shape is None:
         return None
-
-    if isinstance(shape, int) or (isinstance(shape, TensorVariable) and shape.ndim == 0):
+    if isinstance(shape, int) or (isinstance(size, TensorVariable) and shape.ndim == 0):
```

Comment:
I would add parentheses around the or statement conditionsm its not strictly needed but it makes the code intent  much more obvious

Reply:
I don't mind the parantheses, but in the 2nd isinstance you changed shape to size.
This line should have no change at all